themes/bootstrap3/js/record.js

```diff
@@ -34,6 +34,37 @@ function checkRequestIsValid(element, requestType, icon = 'place-hold') {
     });
 }
 
+function wayfinderPlacementLinkLookup(element) {
```

Comment:
I failed to mention this before, but the bootstrap3 theme, including record.js, comes from upstream VuFind distributions. Any changes must be done in the finna2 theme instead.